cmd/login/runner.go

```diff
@@ -297,10 +297,21 @@ func (r *runner) loginWithURL(ctx context.Context, path string, firstLogin bool,
 
 	contextName := kubeconfig.GenerateKubeContextName(i.Codename)
 	if firstLogin {
-		fmt.Fprintf(r.stdout, "A new kubectl context has been created named '%s' and selected.", contextName)
-		fmt.Fprintf(r.stdout, " ")
+		if r.flag.KeepContext {
+			fmt.Fprintf(r.stdout, "A new kubectl context has been created named '%s'.", contextName)
```

Comment:
I notice that our wording is slightly awkward here. WDYT?

```suggestion
			fmt.Fprintf(r.stdout, "A new kubectl context '%s' has been created.", contextName)
```

Should be changed below accordingly.